### PR TITLE
Add a Lessons page

### PR DESCRIPTION
**Pull request by sweisz**
- Header
- List Items
- Button

### CODE REVIEW

File: components/Lessons/Lessons.module.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
-.main {
+.container {
   background-color: white;
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  align-content: start;
+  height: 100vh;
```
Comment (phlgr) [inline components/Lessons/Lessons.module.css]:
```suggestion
  height: 100%;
```
nesting vhs is not always a good idea because this can create unwanted overflows

File: components/Lessons/Lessons.module.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
-.main {
+.container {
   background-color: white;
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  align-content: start;
```
Comment (phlgr) [inline components/Lessons/Lessons.module.css]:
```suggestion
```
This is the default value

File: components/Lessons/Lessons.module.css [hunk 3]
```diff
@@ -1,3 +1,10 @@
-.main {
+.container {
   background-color: white;
```
Comment (phlgr) [inline components/Lessons/Lessons.module.css:2]:
```suggestion
```
This is also the default color

File: components/Lessons/Lessons.module.css [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  align-content: start;
+  height: 100vh;
+}
+.footer {
+  color: red;
 }
```
Comment (phlgr) [inline components/Lessons/Lessons.module.css]: Why is this here?

File: components/Lessons/Lessons.tsx [hunk 5]
```diff
@@ -1,15 +1,28 @@
...
 function Lessons() {
   return (
-    <div className={styles.background}>
-      <LessonsHead label={"Beginner"} link={"/beginner"} />
-      <ListItems title={"Stairway To Heaven - Led Zeppelin"} checkBox={false} />
-      <LessonsButton label={"Beginner"} href={"/lessons/beginner"} />
+    <div className={styles.container}>
+      <div>
+        <LessonsHead label={"Beginner"} link={"/beginner"} />
+      </div>
```
Comment (phlgr) [inline components/Lessons/Lessons.tsx]: These `div` don't seem like they do anything.

File: components/LessonsButton/LessonsButton.tsx [hunk 6]
```diff
@@ -12,13 +12,11 @@ function LessonsButton({ label, href }: LessonsButtonProps) {
...
-      <Link href={btnHref}>
-        <div className={styles.btn}>
-          <a>{btnLabel}</a>
-        </div>
-      </Link>
-    </div>
+    <Link href={btnHref}>
+      <div className={styles.btn}>
+        <a>{btnLabel}</a>
+      </div>
```
Comment (phlgr) [inline components/LessonsButton/LessonsButton.tsx]: This `a` tag should be outside the div element

File: components/ListItems/ListItem.module.css [hunk 7]
```diff
@@ -1,7 +1,7 @@
 .listItem {
   background: var(--list-item-background);
   color: var(--list-item-color);
-  border-top: 1px solid var(--list-item-border-color);
+  border-top: 1px solid #ddd;
```
Comment (phlgr) [inline components/ListItems/ListItem.module.css]:
```suggestion
  border-top: 1px solid var(--list-item-border-color);
```
Return to the var and then change the var itself

**Review by phlgr — CHANGES_REQUESTED** [review]
GJ 🚀 
There seems to be another div around the main element which can also be removed. The main element also has different attributes which are not necessary

File: components/Lessons/Lessons.module.css (hunk 3 above)
Comment (sweisz) [reply components/Lessons/Lessons.module.css:2]: I need this color, because of the background gradient.

File: components/Lessons/Lessons.module.css (hunk 1 above)
Comment (sweisz) [reply components/Lessons/Lessons.module.css]: Does not seems to work here with „%".
